src/wallet/wallet.cpp

```diff
@@ -741,6 +742,11 @@ void CWallet::AddToSpends(const uint256& wtxid)
             AddToSpends(nullifier, wtxid);
         }
     }
+    // TODO: do we want Sapling nullifiers to be stored with Sprout nullifiers?
+    // Sapling spends
+    for (const SpendDescription &spend : tx.vShieldedSpend) {
+        AddToSpends(spend.nullifier, wtxid);
```

Comment:
This needs to be `AddToSaplingSpends()`, and we need a `mapTxSaplingNullifiers` it stores them in (because Sprout and Sapling nullifiers are in different domains).